### PR TITLE
Add DocuSign models

### PR DESCRIPTION
**Pull request by kberzinch**
Add models required for DocuSign ingestion

### CODE REVIEW

**Review by evan10s — APPROVED** [review]
Seems reasonable

File: app/Nova/Actions/CreateFundingAllocationLinesFromJacketPages.php [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+    /**
+     * Get the fields available on the action.
+     *
+     * @return array<\Laravel\Nova\Fields\Field>
+     */
+    public function fields(NovaRequest $request): array
+    {
+        return [
+            Textarea::make('Lines from JacketPages')
```
Comment (JoshOldenburg) [inline app/Nova/Actions/CreateFundingAllocationLinesFromJacketPages.php:102]: Help text would be nice

**Review by JoshOldenburg — APPROVED** [review]
Seems sane

File: app/Nova/Actions/CreateFundingAllocationLinesFromJacketPages.php (hunk 1 above)
Comment (kberzinch) [reply app/Nova/Actions/CreateFundingAllocationLinesFromJacketPages.php:102]: I think it'll be fine, only two-ish people will use this in a year

**Pull request merged**